pkg/secretless/log/doc.go

```diff
@@ -0,0 +1,35 @@
+/*
+Package log provides a type similar to the standard library's "log.Logger".  It allows you to
+log events to stdout and stderr.  It respects Secretless's command line `debug` flag,
+so that calling Debugf or Infof will do nothing unless you started Secretless in
+debug mode.
+
+Your Secretless plugin code should never use the "fmt" or "log" packages or write
+directly to stdout or stderr.  Instead, call Logger() on your connector.Resources
+to get a secretless.Logger as defined below.
+
+The 3 Debug methods and the 3 Info methods do nothing unless you started
+Secretless with the Debug command line flag set to "true".  If you did start
+Secretless in debug mode, they write to stdout.
```

Comment:
I looked over the file, and I _think_ my use is consistent. It's a little hard to tell, though.

godocs don't support code snippets inline with other text, so there is a lot of stuff here that would've been formatted as code that instead we're leveraging capitalization to indicate we're referring to method names.

Reply:
I see what you mean - that's a perfectly good reason to use initial caps. I think it looked good otherwise and I'd agree with keeping the initial cap D here then.